### ISSUE TITLE
Add callback cli command

### ISSUE THREAD

**Issue by dfarr** [resonatehq/resonate#435]
Create a callback associated with a promise.

Commands:
- create

Examples:
```
resonate callbacks create --promise-id foo --root-promise-id bar --timeout 1h --recv default
```

A receiver can be either a string or an object, an object must always have a type and data.
- logical as string (eg: "default")
- physical as url string (eg: "poll://default/6fa89b7e-4a56-40e8-ba4e-78864caa3278")
- physical as object (eg: `{"type": "poll", "data": {"group": "default", "id": "6fa89b7e-4a56-40e8-ba4e-78864caa3278"}}`

See [promises](https://github.com/resonatehq/resonate/tree/main/cmd/promises) and [schedules](https://github.com/resonatehq/resonate/tree/main/cmd/schedules) for a reference.

**Comment by Madraceee** [comment]
Hey, this project seems interesting. Can I take up this issue to start contributing to this repo?

**Comment by dfarr** [comment]
Of course! Thanks so much @Madraceee, let me know if I can help

**Comment by Madraceee** [comment]
I will go through the repo and start working asap!!

**Comment by Madraceee** [comment]
@dfarr If my understanding is right , I have to make a new command which calls the callback functions from "pkg/client/v1/v1.go" ?

**Comment by dfarr** [comment]
Yes that's correct! The parameters to that call correspond to the flags the command should expose, as detailed above, the only tricky parameter is recv which can be either a string or an object as detailed above.

**Comment by Madraceee** [comment]
Got it 
I will work on this

**Comment by Madraceee** [comment]
I have two doubts

1. What should be the contentType when sending default or a url?

**Comment by Madraceee** [comment]
Other than the contentType part,
I have written the code. I am still learning the whole functionality, so can you look at my code and let me know if any changes has to be done?
I can push the changes if you are okay with it.

**Comment by TusharMohapatra07** [comment]
Is this issue still open for contributions ? @dfarr @Madraceee

**Comment by Madraceee** [comment]
@TusharMohapatra07 I have written the code, i just need clarifications from the team

**Comment by dfarr** [comment]
Awesome @Madraceee! Please open a PR and we will review :)

**Comment by flossypurse** [comment]
@Madraceee - just following up here on how we can help.

What do you need help clarifying?
We have a new channel in Discord dedicated to resonate engineering discussions (#resonate-engineering): https://resonatehq.io/discord

Feel free to hop in there to chat with us in realtime!

**Comment by flossypurse** [comment]
Oh, sorry @Madraceee - seeing your PR here: https://github.com/resonatehq/resonate/pull/446
